gui/src/assets/js/networkInspect.js

```diff
@@ -21,7 +21,7 @@ function waitingConnected(promise, interval, cancel, timeout) {
             }
           },
           () => {
-            if (res.data.message !== "the last request is being processed") {
+            if (res.data.message !== "Processing last request") {
```

Comment:
I think the expression is not intuitive enough and may cause misunderstanding. What do you think?